crates/bevy_gizmos/src/gizmos.rs

```diff
@@ -303,6 +304,56 @@ impl<'s> Gizmos<'s> {
         self.add_list_color(color, 24);
     }
 
+    /// Draw a bounding box.
+    ///
+    /// # Example
+    /// ```
+    /// # use bevy_gizmos::prelude::*;
+    /// # use bevy_render::{prelude::*, primitives::Aabb};
+    /// # use bevy_math::prelude::*;
+    /// # use bevy_transform::prelude::*;
+    /// fn system(mut gizmos: Gizmos) {
+    ///     let aabb = Aabb::from_min_max(Vec3::NEG_ONE, Vec3::ONE);
+    ///     gizmos.bounding_box(aabb, Transform::IDENTITY, Color::GREEN);
+    /// }
+    /// # bevy_ecs::system::assert_is_system(system);
+    /// ```
+    #[inline]
+    pub fn bounding_box(&mut self, aabb: Aabb, transform: impl TransformPoint, color: Color) {
```

Comment:
Can't you call the `cuboid` method internally? This would just require computing an alternative center right?

Reply:
It was slightly more complicated than that because I wanted the bounding boxes to respect shearing as well :)

Reply:
I would expect an _Axis Aligned_ bounding box to not be rotated or sheared :P.

Bevy doesn't define transform operations over AABB (outside of `scene_viewer/main.rs`) so the result of transforming AABB is unspecified yet.

- If rotating an AABB should result in a cuboid, then I don't see the interest in specifying rotation of AABBs independently of cuboids.
- If rotating an AABB should result in another AABB, then a simple transform is not enough, it would need something like what is done in `scene_viewer/main.rs`.

Reply:
No it's fine as is. Well, I can see good reasons for both choices. Of course you want the displayed bounding boxes (which as displayed are technically not Aabbs) to follow as closely what bevy uses for frustrum culling, so it makes sense to show rotated Aabbs for the bounding box debug plugin.